G1ANT.Addon.Xlsx/G1ANT.Addon.Xlsx/Commands/XlsxCloseCommand.md

```diff
@@ -1,50 +1,35 @@
+| `if`           | [bool](G1ANT.Language/G1ANT.Language/Structures/BooleanStructure.md) | no       | true                                                        | Executes the command only if a specified condition is true   |
+| `timeout`      | [timespan](G1ANT.Language/G1ANT.Language/Structures/TimeSpanStructure.md) | no       | [♥timeoutcommand](G1ANT.Language/G1ANT.Addon.Core/Variables/TimeoutCommandVariable.md) | Specifies time in milliseconds for G1ANT.Robot to wait for the command to be executed |
+| `errorcall`    | [procedure](G1ANT.Language/G1ANT.Language/Structures/ProcedureStructure.md) | no       |                                                             | Name of a procedure to call when the command throws an exception or when a given `timeout` expires |
+| `errorjump`    | [label](G1ANT.Language/G1ANT.Language/Structures/LabelStructure.md) | no       |                                                             | Name of the label to jump to when the command throws an exception or when a given `timeout` expires |
+| `errormessage` | [text](G1ANT.Language/G1ANT.Language/Structures/TextStructure.md) | no       |                                                             | A message that will be shown in case the command throws an exception or when a given `timeout` expires, and no `errorjump` argument is specified |
+| `errorresult`  | [variable](G1ANT.Language/G1ANT.Language/Structures/VariableStructure.md) | no       |                                                             | Name of a variable that will store the returned exception. The variable will be of [error](G1ANT.Language/G1ANT.Language/Structures/ErrorStructure.md) structure  |
 
-For more information about `if`, `timeout`, `errorjump` and `errormessage` arguments, please visit [Common Arguments](https://github.com/G1ANT-Robot/G1ANT.Manual/blob/master/G1ANT-Language/Common-Arguments.md)  manual page.
+For more information about `if`, `timeout`, `errorcall`, `errorjump`, `errormessage` and `errorresult` arguments, see [Common Arguments](G1ANT.Manual/appendices/common-arguments.md) page.
 
-This command is contained in **G1ANT.Addon.Xlsx.dll**.
-See: [https://github.com/G1ANT-Robot/G1ANT.Addon.Xlsx](https://github.com/G1ANT-Robot/G1ANT.Addon.Xlsx)
+## Example
 
-**Example 1:**
-
-Document with .xlsx file extension, which was opened as first one, is now being closed.
-
-```G1ANT
-xlsx.close
-```
-
-**Example 2:**
-
-Here document with .xlsx file extension with ID '444' is being closed.
+In the script below, two Excel files are opened (be sure to provide real filepaths) and their IDs are stored in variables specified by the `result` argument. Then the second file is closed — if its ID wasn’t specified, the `xlsx.close` command would close the first file:
 
 ```G1ANT
-xlsx.close id ♥444
+xlsx.open path C:\Tests\Book1.xlsx result ♥xlsId1
+xlsx.open path C:\Tests\Book2.xls result ♥xlsId2
+xlsx.close id ♥xlsId2
```

Comment:
xlsx.open path ♥environment⟦TEMP⟧\Book1.xlsx createifnotexist true result ♥xlsId1
xlsx.open path ♥environment⟦TEMP⟧\Book2.xls createifnotexist true result ♥xlsId2
xlsx.close id ♥xlsId2
xlsx.close id ♥xlsId1

Reply:
If we use ♥environment⟦TEMP⟧ in examples its going to work just after copy paste.